api/mysagw/caluma_document_parser.py

```diff
@@ -177,23 +177,22 @@ def _handle_files(self, question, answer):
             }
 
         filename_list = []
+        page_count = len(self.merger.pages)
         for value in answer["node"][
             self.value_key_for_question(question["__typename"])
         ]:
             name = value["name"]
-            if (value.get("metadata", {}) or {}).get(
-                "content_type",
+            if (
+                content_type := (value.get("metadata", {}) or {}).get(
+                    "content_type",
+                )
             ) in SUPPORTED_MERGE_CONTENT_TYPES:
```

Comment:
```suggestion
            content_type = (value.get("metadata", {}) or {}).get(
                "content_type"
            )
            if content_type in SUPPORTED_MERGE_CONTENT_TYPES:
```

Reply:
Was actually thinking about this :sweat_smile:  Done :+1: